modules/ROOT/pages/security.adoc

```diff
@@ -92,7 +92,9 @@ include::partial$security/sanitizing-html-input-and-protecting-against-xss-attac
 [[securing-embedded-external-resources]]
 === Securing embedded external resources
 
-include::partial$security/securing-embedded-external-resources.adoc[]
+include::partial$configuration/sandbox_iframes_options.adoc[]
+
+include::partial$configuration/convert_unsafe_embeds_options.adoc[]
```

Comment:
```suggestion
include::partial$configuration/sandbox_iframes.adoc[]

include::partial$configuration/convert_unsafe_embeds.adoc[]
```
Looks much better, these should just be named after the options. Also, is there an existing partial for `sandbox_iframes_exclusions` or is that only in a separate PR?

Reply:
No I just created them, I'll fix in this PR